life_expectancy/cleaning.py

```diff
@@ -21,93 +20,134 @@
 
 # Define variables cleaning  and filtering data
 COMPOSED_COL = 'unit,sex,age,geo\\time'
-DECOMPOSED_COLs = ['unit', 'sex', 'age','region']
+DECOMPOSED_COLS= ['unit', 'sex', 'age','region']
```

Comment:
This is a good practice! You might have to alter it a bit for assignment 4, but refactoring code is also a good practice and  it's better than over-engineering. 👍